lib/workers/repository/init/index.spec.ts

```diff
@@ -29,5 +29,16 @@ describe(getName(__filename), () => {
       const renovateConfig = await initRepo({});
       expect(renovateConfig).toMatchSnapshot();
     });
+    it('warns on unsupported options', async () => {
+      apis.initApis.mockResolvedValue({} as never);
+      onboarding.checkOnboardingBranch.mockResolvedValueOnce({});
+      config.getRepoConfig.mockResolvedValueOnce({
+        filterOutUnavailableUsers: true,
+      });
+      config.mergeRenovateConfig.mockResolvedValueOnce({});
+      secrets.applySecretsToConfig.mockReturnValueOnce({} as never);
+      await initRepo({});
+      expect(logger.logger.warn).toHaveBeenCalled();
```

Comment:
Thinking if we should match called args to not silently failed in future if there are more warnings logged.

Reply:
Yes, that makes total sense. Done with cbd5430df8194705e76803520f824004d78839b6.